qiita_pet/handlers/api_proxy/processing.py

```diff
@@ -62,7 +64,8 @@ def list_commands_handler_get_req(artifact_types):
     artifact_types = artifact_types.split(',')
     cmd_info = [
         {'id': cmd.id, 'command': cmd.name, 'output': cmd.outputs}
-        for cmd in Command.get_commands_by_input_type(artifact_types)]
+        for cmd in Command.get_commands_by_input_type(
+            artifact_types, exclude_analysis=exclude_analysis)]
```

Comment:
The "arbitrary" partitioning is matching the "upstream" and "downstream" commands that has been in Qiime for a long time. i.e. commands that you need to run to get a BIOM table, and commands that you run in a BIOM table to gain insight.